storage/postgres/notificator.go

```diff
@@ -42,8 +42,8 @@ const (
 )
 
 type Notificator struct {
+	isListening bool
```

Comment:
why remove the atomic assignment?

Reply:
It is not necessary as isListening is only used under connectionMutex.Lock